xapian-applications/omega/templates/query

```diff
@@ -106,7 +106,7 @@ $if{$field{size},<span title="$html{$field{size}} bytes">Size: <b>$html{$filesiz
 </div>
 </div>
 </td>
-<td><b><a href="$html{$field{url}}">$html{$or{$field{caption},$field{title},$prettyurl{$field{url}},Untitled}}</a></b><br>
+<td><b><a href="$html{omega.cgi?FMT=$url{clicklog}&QUERYID=$url{$hash{md5,$transform{s/^\s+|\s+$$//g,\1,$query}}}&DOCID=$url{$id}&URL=$url{$field{url}}}">$html{$or{$field{caption},$field{title},$prettyurl{$field{url}},Untitled}}</a></b><br>
```

Comment:
You do the selection the same way `inc/anyall*` do — accept a default, and let the user change that if they want to switch the other behaviour on. (And then this should be documented — note that it isn't for the previous stuff, so it doesn't really matter where you put it. We should at some point flesh this out into a proper manual for omega, but that's not part of this PR or even project!)

If we wanted it easier to configure, we could use an environment variable (`$env{VAR}`), or possibly using `$opt{}` and `$set{}` (so you'd just set the behaviour you want at the start of the omegascript).